extension/src/plots/model/collect.ts

```diff
@@ -205,10 +211,26 @@ const getMultiImageInd = (path: string) => {
   return Number(fileName)
 }
 
+const collectImageBoundingBoxes = (
+  boundingBoxes: { label: string; box: BoundingBox }[]
+): { [label: string]: BoundingBox[] } => {
+  const acc: { [label: string]: BoundingBox[] } = {}
+
+  for (const { label, box } of boundingBoxes) {
+    if (!acc[label]) {
+      acc[label] = []
```

Comment:
You don't need the box in the array yet?

Reply:
Apologies, I'm not sure what you mean 🤔 Are you saying we should just leave the `comparisonData` boxes as `{ label: string, box: BoundingBox }[]`, and only convert them to the new format (`{ label: BoundingBox[] }` when we are grabbing the selected plot data?

Reply:
Below, you do `acc[label].push(box)`, but here you initialize an empty array. What is the reason for not doing `acc[label] = [box]`

Reply:
> Below, you do acc[label].push(box), but here you initialize an empty array. What is the reason for not doing acc[label] = [box]

Oh, ok, thanks for the clarification! Looking at the two methods for collection one or more boxes: 

```js
  for (const { label, box } of boundingBoxes) {
    if (!acc[label]) {
      acc[label] = [box]
      continue;
    }

    acc[label].push(box)
```

```js
  for (const { label, box } of boundingBoxes) {
    if (!acc[label]) {
      acc[label] = []
    }

    acc[label].push(box)
```

I didn't think there was much of a difference, but I'm happy to update if you prefer the first example!